src/subsplit_dag.hpp

```diff
@@ -60,6 +60,27 @@ class SubsplitDAG {
     SizeVector new_node_ids, new_edge_idxs, node_reindexer, edge_reindexer;
   };
 
+  // Nearest Neighbor Interchange Operations
+  using NNIOperation = std::pair<Bitset, Bitset>;
+  using ListOfNNIs = std::set<NNIOperation>;
```

Comment:
In my mind, lists and sets are different.

Reply:
Name updated to reflect sets.